configure.ac

```diff
@@ -436,13 +436,15 @@ if test "x$with_bz2lib" != "xno"; then
   esac
 fi
 
+PKG_PROG_PKG_CONFIG
```

Comment:
This looks reasonable to me.

Heads up for @mmatuska that this macro might be missing when building distributions, it's distributed with pkg-config and doesn't appear to be included in the typical autoconf packages.